kivy/core/window/__init__.py

```diff
@@ -1984,9 +1984,15 @@ def on_keyboard(self, key, scancode=None, codepoint=None,
                            "semantics.")
 
         # Quit if user presses ESC or the typical OSX shortcuts CMD+q or CMD+w
+        # On Android a back key/gesture is mapped to 27 and initiates a pause.
+        # Consume the event and tell Android to pause.
         # TODO If just CMD+w is pressed, only the window should be closed.
         is_osx = platform == 'darwin'
-        if WindowBase.on_keyboard.exit_on_escape:
+        if key == 27 and platform == 'android':
+            from android import mActivity
+            mActivity.moveTaskToBack(True)
+            return True
+        elif WindowBase.on_keyboard.exit_on_escape:
             if key == 27 or all([is_osx, key in [113, 119], modifier == 1024]):
```

Comment:
just question:
shouldnt we do some key mapper to constants for keys such "27" (android) and 113 like here etc?

Reply:
No, because it is a change that would be mission creep for this PR. This PR uses the existing conventions for the express purpose of being clear about the functional changes. 

If this kind of thing is important to you, please submit a PR.
- Explain why you propose the changes.
- Show that the changes are consistent across the Kivy code base.
- Think about 27 which is not ESC on Android, it just looks that way to a user because of this https://github.com/kivy/kivy/blob/master/kivy/core/window/window_sdl2.py#L209-L211

Reply:
I was asking to make it in this PR :) but just asked question globally, in future [separate PR], whatever :)

But you are maybe right. I was thinking in same way pygame was handling keys, they were mapped like K_BACKSPACE, K_ESCAPE, K_SLASH etc.

I though it might improve readability of code somehow